### PR TITLE
Add Contributor Section

### PR DESCRIPTION
**Pull request by souvikpramanikgit**
I added the contributor section.

### CODE REVIEW

**Comment by souvikpramanikgit** [comment]
Fix #21

**Comment by c0sm0void** [comment]
@souvikpramanikgit please resolve the conflicts and do a few more updates as per your issue, like animations and layout fixes. Someone else has already done the contributor view.

File: README.md [hunk 1]
```diff
@@ -105,10 +105,20 @@ ReVot/
...
+
+Big thanks to all the contributors! 🎉
+
+<a href="https://github.com/c0sm0void/ReVot/pulse"> <img align="center" src="https://contrib.rocks/image?max=100&repo=c0sm0void/ReVot" /> </a> 
+
+<br>
 
-### License 📜
+## License 📜
 This project is licensed under the [MIT License](https://opensource.org/licenses/MIT).
```
Comment (whyvineet) [inline README.md:119]: This paragraph is showing with H2 text font

File: README.md [hunk 2]
```diff
@@ -122,4 +132,4 @@ You are free to use, modify, and distribute this software as long as the origina
   <a href="https://github.com/c0sm0void/ReVot">
```
Comment (whyvineet) [inline README.md:132]: Explain this as this is showing as it is in the bottom

**Review by whyvineet — CHANGES_REQUESTED** [review]
I noticed a few areas where the README could be improved for clarity and consistency. These changes would help to ensure that the README is easy for potential contributors to understand and follow. Thank you for your work on this project!

**Review by c0sm0void — APPROVED** [review]

**Pull request merged**